machine.go

```diff
@@ -4,12 +4,14 @@
 package gomaasapi
 
 import (
+	"fmt"
 	"net/http"
 
```

Comment:
Did we decide to use :x: to mark "Fixed"? Now it looks weird to me and I think I'd rather use :+1: instead